internal/api/auth/context.go

```diff
@@ -59,6 +59,10 @@ func VerifyTokenAndWriteCtxData(ctx context.Context, token, orgID string, t Toke
 	return context.WithValue(ctx, dataKey, CtxData{UserID: userID, OrgID: orgID, ProjectID: projectID, AgentID: agentID}), nil
 }
 
+func GetCtxDataKey() key {
+	return dataKey
+}
```

Comment:
i believe we have done this in the old repo as well, but won't this "compromise" the key? @adlerhurst

Reply:
if we make a setcontextdata func i dont need the key